### ISSUE TITLE
Uninstall fails to uninstall

### ISSUE THREAD

**Issue by CelsoSantos** [FiloSottile/mkcert#355]
When trying to uninstall the certificates, `mkcert` fails with the following message

```
ERROR: failed to execute "security remove-trusted-cert": exit status 1

SecTrustSettingsRemoveTrustSettings: No Trust Settings were found.
```

Here's the shell output:

```
$ mkcert -CAROOT
/Users/zed/Library/Application Support/mkcert

$ mkcert -uninstall
ERROR: failed to execute "security remove-trusted-cert": exit status 1

SecTrustSettingsRemoveTrustSettings: No Trust Settings were found.

$ sudo mkcert -uninstall
ERROR: failed to execute "security remove-trusted-cert": exit status 1

SecTrustSettingsRemoveTrustSettings: No Trust Settings were found.

```

**Comment by felipevalois** [comment]
Any luck with that?

**Comment by CelsoSantos** [comment]
No... I'm still waiting on a response

**Comment by 5265949743** [comment]
$ mkcert -CAROOT
/Users/zed/Library/Application Support/mkcert

$ mkcert -uninstall
ERROR: failed to execute "security remove-trusted-cert": exit status 1

SecTrustSettingsRemoveTrustSettings: No Trust Settings were found.

$ sudo mkcert -uninstall
ERROR: failed to execute "security remove-trusted-cert": exit status 1

SecTrustSettingsRemoveTrustSettings: No Trust Settings were found.

**Comment by 5265949743** [comment]
$ mkcert -CAROOT
/Users/zed/Library/Application Support/mkcert

$ mkcert -uninstall
ERROR: failed to execute "security remove-trusted-cert": exit status 1

SecTrustSettingsRemoveTrustSettings: No Trust Settings were found.

$ sudo mkcert -uninstall
ERROR: failed to execute "security remove-trusted-cert": exit status 1

SecTrustSettingsRemoveTrustSettings: No Trust Settings were found.

**Comment by CelsoSantos** [comment]
Just an update: I revisited this issue today and it *might* be a non-issue.
The command to uninstall only fails with this message when it has already been uninstalled before.

If one executes `--install` before `--uninstall`, the uninstall process goes OK. However, even after uninstall the rootCA files persist on /Library/Application\ Support/mkcert and I feel like they should be deleted when the uninstall command is executed.
It's also not entirely clear if uninstalling will actually remove the certificate from the keystores to which it was added upon request to generate a new one.

Could the uninstall process be a bit more detailed @FiloSottile ? I'd be available to help document it
